aws/resource_aws_network_acl_rule_test.go

```diff
@@ -162,20 +178,15 @@ func TestAccAWSNetworkAclRule_ipv6VpcAssignGeneratedIpv6CidrBlockUpdate(t *testi
 		Steps: []resource.TestStep{
 			{
 				Config: testAccAWSNetworkAclRuleConfigIpv6VpcAssignGeneratedIpv6CidrBlockUpdate(false),
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckVpcExists(vpcResourceName, &vpc),
-					resource.TestCheckResourceAttr(vpcResourceName, "assign_generated_ipv6_cidr_block", "false"),
-					resource.TestCheckResourceAttr(vpcResourceName, "ipv6_cidr_block", ""),
-				),
 			},
 			{
 				Config: testAccAWSNetworkAclRuleConfigIpv6VpcAssignGeneratedIpv6CidrBlockUpdate(true),
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckVpcExists(vpcResourceName, &vpc),
-					resource.TestCheckResourceAttr(vpcResourceName, "assign_generated_ipv6_cidr_block", "true"),
-					resource.TestMatchResourceAttr(vpcResourceName, "ipv6_cidr_block", regexp.MustCompile(`/56$`)),
```

Comment:
This testing might seem a little odd, but it is written this way to ensure assumptions about changes to an upstream resource have the desired outcome in the tested resource and the `Check`s should stay. Please see the reference issue: https://github.com/terraform-providers/terraform-provider-aws/issues/6710